src/directory-sync/interfaces/user.interface.ts

```diff
@@ -20,6 +20,7 @@ export interface User<
   }[];
   username: string;
   last_name: string;
+  job_title: string;
```

Comment:
Do we not need any "setters" for these fields in Node - is it smart enough to just extract them from the payload auto-magically? I'm assuming so since the tests pass 😅, but just wanted to confirm (since I thought I'd find something like https://github.com/workos/workos-ruby/pull/178/files#diff-df42219ae143db61d39dd9502412fe14ef802a013750d3f599395b19540e3056R27 in here too)

Reply:
The underlying objects used by the SDK are just Plain Old JavaScript Objects (POJOs), so the typings here are just applied on top of the object (which may not actually match what is there at runtime).

So no, we don't need to specify any "setters".